packages/loot-design/src/components/modals/SelectLinkedAccounts.js

```diff
@@ -1,254 +1,200 @@
-import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
 
-import { View, Modal, P, Button, Strong, CustomSelect } from '../common';
+import { colors } from '../../style';
+import { View, Modal, Button, Text } from '../common';
+import Autocomplete from '../NewAutocomplete';
+import { TableHeader, Table, Row, Field } from '../table';
 
-function EmptyMessage() {
-  return null;
-}
+const addAccountOption = { value: 'new', label: 'Create new account' };
 
 export default function SelectLinkedAccounts({
-  upgradingAccountId,
```

Comment:
I tried figuring out why we need this `upgradingAccountId` prop.. it seems it's only used to display a slightly different text label? So possibly we can remove it in other code paths too? What do you think?
cc @fstybel

Reply:
I'm guessing it's for adding syncing to an existing account.